Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -450,6 +450,8 @@ extension TopTabsViewController {
         self.flushPendingChanges()
         UIView.animate(withDuration: TopTabsUX.AnimationSpeed, animations: {
             self.collectionView.reloadData()
+            // Workaround - reloadData() is not triggering collectionView(_:cellForItemAt:)
+            self.collectionView.reloadItems(at: self.collectionView.indexPathsForVisibleItems)
```

Comment:
reloading all the items is something I tried very hard to avoid in TopTabs. Is there another way of figuring out which tab might have been selected previously without having to resort to reloading all visible?